Add inverted RFLink cover description (#10177)
home-assistant/home-assistant.github.io: source/_components/cover.rflink.markdown

          description: The `aliases` which do not respond to group commands.
          required: false
          type: [list, string]
        type:
          description: The option to invert (`inverted`) on/off commands sent to the RFLink device or not (`standard`).
          required: false

>           default: 

Reviewer: Is there a default? In the code I see `[]`, but otherwise we can leave it away

>           description: Set to `standard` or `inverted`. `inverted` will invert the on/off commands sent to the RFLink device, `standard` will not invert the on/off commands sent to the RFLink device. When omitted, the commands to the RFLink device will be inverted if the ID of the device starts with 'newkaku'.

Reviewer: I think that the description could be shorter

Author: I agree, too many words in the description field. Am I allowed to refer to a configuration example in another section to elaborate on the type property?

Author: I shortened it and hope it is still proper English.

Reviewer: ```suggestion
          description: The option to invert (`inverted`) on/off commands sent to the RFLink device or not (`standerd`).
```

This is a suggestion. If I understand correctly, the code will automatically invert when you start using the KAKU device? That is something important to mention elsewhere in the text 😉

Author: It will only invert if the device ID starts with 'newkaku', those are the ASUN-650 devices. I've mentioned it with the example I provided.

Author: I applied your suggestion.

          type: [list, string]
{% endconfiguration %}

### Setting up a KAKU ASUN-650 device

In RFLink, the ON and DOWN command are used to close the cover and the OFF and UP command are used to open the cover. The KAKU (COCO) ASUN-650 works the other way around, it uses the ON command to open the cover and the OFF command to close the cover. The RFLink cover component has an extra property named `type` that takes 2 values:
- standard: Do not invert the on/off commands sent to the RFLink device.
- inverted: Invert the on/off commands sent to the RFLink device.

The following configuration example shows how to use the `type` property:

```yaml
# Example configuration.yaml entry that shows how to
# use the type property.

> cover rflink:

Reviewer: ```suggestion
cover:
```

  - platform: rflink

>     device_defaults:

Reviewer: You can leave this away by default it's `false`

Author: Ok, I removed the complete 'device_defaults' section. I'm not that skilled with Home Assistant and learning everyday ;-)
